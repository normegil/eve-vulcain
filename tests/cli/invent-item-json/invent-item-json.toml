bin.name = "eve-vulcain"
args = "invent item \"Damage Control II\" --strict --json -vvv"
status.code = 0
