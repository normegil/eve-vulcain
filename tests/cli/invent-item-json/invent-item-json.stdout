{"searched_item_name":"Damage Control II","facilities":[{"facility_name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","costs":{"inputs":{"inputs":[{"name":"Datacore - Nanite Engineering","orders":[{"quantity":2,"price_per_unit":94504.32,"total":189008.64}]},{"name":"Datacore - Molecular Engineering","orders":[{"quantity":2,"price_per_unit":96661.27,"total":193322.54}]}],"total":382331.18000000005},"job_cost":{"estimated_item_value":177538.88791922803,"system_cost_index":0.0014,"facility_tax":0.0025,"total":67.1096996334682,"job_cost_modifier":null},"total_run":382398.28969963355},"time":{"base_time":13800,"job_duration_modifier":null,"advanced_industry_skill_level":5,"total_time_run":11730},"success_probability":{"base_success_chance":0.34,"skills":[["Molecular Engineering",3],["Nanite Engineering",3],["Gallente Encryption Methods",4]],"final_success_chance":0.44200000000000006},"total_cost_normalized":865154.5015828811,"total_time_normalized":26538},{"facility_name":"Maurasi - BPs Center, read BIO","costs":{"inputs":{"inputs":[{"name":"Datacore - Nanite Engineering","orders":[{"quantity":2,"price_per_unit":94504.32,"total":189008.64}]},{"name":"Datacore - Molecular Engineering","orders":[{"quantity":2,"price_per_unit":96661.27,"total":193322.54}]}],"total":382331.18000000005},"job_cost":{"estimated_item_value":177538.88791922803,"system_cost_index":0.0774,"facility_tax":0.002,"total":326.94851443653357,"job_cost_modifier":0.03},"total_run":382658.1285144366},"time":{"base_time":13800,"job_duration_modifier":0.15,"advanced_industry_skill_level":5,"total_time_run":9970},"success_probability":{"base_success_chance":0.34,"skills":[["Molecular Engineering",3],["Nanite Engineering",3],["Gallente Encryption Methods",4]],"final_success_chance":0.44200000000000006},"total_cost_normalized":865742.3722046075,"total_time_normalized":22556}]}
